spec/features/restaurant/user_sees_details_of_restaurant_spec.rb

```diff
@@ -12,6 +12,13 @@
   let!(:restaurant) { FactoryGirl.create(:restaurant) }
   let!(:user) { FactoryGirl.create(:user) }
 
+  before(:each) do
+    50.times { FactoryGirl.create(:review, restaurant_id: restaurant.id) }
+  end
+  after(:each) do
+    Review.delete_all
+  end
```

Comment:
You shouldn't actually need this `after` action - the db should get cleaned after every test!